shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-search.tsx

```diff
@@ -212,33 +213,36 @@ export default () => {
     [traceCount],
   );
 
-  const columns = [
+  const columns: Array<ColumnProps<object>> = [
```

Comment:
record type is object?

Reply:
I also don't know what is in the record in this place. I see the type of record given before is any.

Reply:
I wrote the recordType according to columns.